mne_bids_pipeline/tests/test_run.py

```diff
@@ -278,3 +282,131 @@ def test_missing_sessions(
         print()
         with context:
             main()
+
+
+def test_session_specific_mri(
```

Comment:
I think this should make `doc` not run the test.
```suggestion
@pytest.mark.dataset_test
def test_session_specific_mri(
```
But then you'll want to modify `.github/workflows/run_tests.yml` to have another `pytest` line that *does* run this test.

Reply:
... I think that would help us move toward "CircleCI should run all unit tests that it needs for docs" and "GHA runs non-doc unit tests, including a couple of pipeline caching tests"

Reply:
ok.  IIUC though that would involve downloading the dataset from OSF into the GHA runner. Is that better than setting it up as a CircleCI job that is dependent on `cache-MNE-funloc-data`?  I guess the GHA job would be free, so that's one advantage...

Reply:
Yeah the DL and cache on GHA should be pretty quick assuming it works and doesn't hit a size limit or anything